01-cloudformation/README.md

```diff
@@ -160,6 +160,8 @@ name.
 
 _Why do we prefer the YAML format for CFN templates?_
 
+Answer: More concise, easier to format
+
 #### Question: Protecting Resources
 
 _What else can you do to prevent resources in a stack from being deleted?_
```

Comment:
Other than using Termination Protection, to prevent resources from deletion, you can:
- Set the DeletionPolicy attribute to prevent the deletion of an individual resource at the stack level.
- Use AWS Identity and Access Management (IAM) policies to restrict the ability of users to delete or update a stack and its resources.
- Assign a stack policy to prevent updates to stack resources.